src/main/resources/templates/itemDetail.html

```diff
@@ -6,5 +6,29 @@
 </head>
 <body>
 <h2>别想了 输出都转义了 CSRF都加Token了</h2>
+<input type="hidden" value="这里真的没有sql注入">
```

Comment:
Remove these two lines.

- First line is just a notice that you should check about the data from user input.
- Second line is not a form, please remove.